.github/workflows/notification.yml

```diff
@@ -12,6 +12,7 @@ env:
 
 jobs:
   notify:
+    if: github.event.pull_request.merged == true
```

Comment:
@fposchner-cc We would not need this if condition too !

Reply:
workflow is already implemented by jenkins ci. eg: https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fleanix-microservice-intelligence-plugin/detail/PR-11/2/tests/